README.md

```diff
@@ -1 +1,44 @@
-# codeclimate-grep
\ No newline at end of file
+# Code Climate Grep Engine
+
+[![Code
+Climate](https://codeclimate.com/github/codeclimate/codeclimate-grep/badges/gpa.svg)](https://codeclimate.com/github/codeclimate/codeclimate-grep)
+
+`codeclimate-grep` is a Code Climate engine that finds specified text and gives
+a message.
+
+Example config:
+
+```
+engines:
+  grep:
+    enabled: true
+    config:
+      patterns:
+      - def set_\w+
+      - \.set_\w+
+      output: "Found a deprecated set_ method"
```

Comment:
I agree that we should revisit this before stable but shouldn't block the beta release.